src/quo/foundations/colors.cljs

```diff
@@ -188,6 +188,8 @@
 (def danger "#E95460")
 
 ;; Danger with transparency
+(def danger-opa-5 (alpha danger 0.05))
```

Comment:
we don't need these colors btw, 
instead it's better to use
`(colors/resolve-color :danger theme 5)`

Reply:
Updated